src/main/java/im/status/keycard/KeycardApplet.java

```diff
@@ -847,6 +861,49 @@ private void loadSeed(byte[] apduBuffer) {
     resetKeyStatus();
     JCSystem.commitTransaction();
   }
+  
+  // Load up to 155 bytes of data into a buffer. This will generally be used for Shamir secrets or
+  // Bitcoin multisig scripts. The buffer (`savedData`) does not interact with anything in the applet.
+  // [PIN protected APDU]
+  private void loadData(APDU apdu) {
+    byte[] apduBuffer = apdu.getBuffer();
+    secureChannel.preprocessAPDU(apduBuffer);
```

Comment:
Wait... is this an encrypted APDU? I thought we were just going to leave this as a plain-text data store. I'm not complaining, but it affects our buffer size conversation above ^

Reply:
Yes, encrypted. How does it affect the buffer size?

Reply:
This needs to be an encrypted APDU, because you are potentially using it to store shards of a key.

Reply:
Good point